recipes/delegation/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "delegation" %}
+{% set version = "1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/f8/54/d65a78f7f42aacefe17c1c437ed8e6525d1caed1c899de7f5c724a5037ee/delegation-1.1.zip
+  sha256: 3638afde814f3e0d540556653f2e09a5c7d1701ee782c5d6ba32d7ab396a5408
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - delegation
+
+about:
+  home: https://github.com/symonsoft/delegation
+  license: BSD
+  license_family: BSD
+  license_file: 
+  summary: Delegate pattern
```

Comment:
can we have a slight more detailed description also  for the other pkgs :)

Reply:
lol. Yes, absolutely.